pkg/pool-utils/contracts/factories/FactoryWidePauseWindow.sol

```diff
@@ -15,25 +15,46 @@
 pragma solidity ^0.7.0;
 pragma experimental ABIEncoderV2;
 
+import "@balancer-labs/v2-interfaces/contracts/solidity-utils/helpers/BalancerErrors.sol";
+
+import "@balancer-labs/v2-solidity-utils/contracts/helpers/TemporarilyPausable.sol";
+
 /**
  * @dev Utility to create Pool factories for Pools that use the `TemporarilyPausable` contract.
  *
  * By calling `TemporarilyPausable`'s constructor with the result of `getPauseConfiguration`, all Pools created by this
  * factory will share the same Pause Window end time, after which both old and new Pools will not be pausable.
  */
-contract FactoryWidePauseWindow {
+abstract contract FactoryWidePauseWindow {
```

Comment:
Why is this `abstract`?

Reply:
Would this ever be instantiated directly? Just seemed like it should be an abstract base contract. Anyway I can put it back.